src/components/custom-tree-item.jsx

```diff
@@ -41,9 +72,12 @@ const CustomContent = React.forwardRef(function CustomContent(props, ref) {
                 expanded && styles.expanded,
                 selected && styles.selected,
                 focused && styles.focused,
-                disabled && styles.disabled
+                disabled && styles.disabled,
+                hover && { backgroundColor: `${theme.aggrid.highlightColor} !important`, borderRadius: '16px' } // Add hover style with  background color
```

Comment:
Adding an aggrid css color to a TreeItem seems a bad idea

Reply:
I agree that using aggrid css is not ideal but since the hover and aggrid will always be the same color, I tought of using it instead of creating another variable with the same value. But I can change it.